langchain4j-core/src/test/java/dev/langchain4j/model/chat/common/ChatLanguageModelCapabilities.java

```diff
@@ -0,0 +1,44 @@
+package dev.langchain4j.model.chat.common;
+
+import static java.util.Objects.nonNull;
+
+import dev.langchain4j.model.chat.ChatLanguageModel;
+
+public class ChatLanguageModelCapabilities extends ChatModelCapabilities<ChatLanguageModel> {
```

Comment:
```suggestion
public class ChatModelAndCapabilities extends ChatModelCapabilities<ChatLanguageModel> {
```
WDYT? (ChatLanguageModel will be renamed into ChatModel soon)